webapi/src/main/scala/org/knora/webapi/slice/admin/domain/service/ProjectExportService.scala

```diff
@@ -123,37 +121,24 @@ final case class ProjectExportServiceLive(
   private val triplestoreService: TriplestoreService
 ) extends ProjectExportService {
 
-  override def exportProjectTriples(project: KnoraProject): Task[Path] = {
-    val tempDir    = Files.createTempDirectory(project.shortname)
-    val targetFile = tempDir.resolve(project.shortname + ".trig")
-    exportProjectTriples(project: KnoraProject, targetFile)
-  }
+  override def exportProjectTriples(project: KnoraProject): Task[Path] =
+    Files
+      .createTempDirectory(Some(project.shortname), fileAttributes = Nil)
+      .map(trigExportFile(project, _))
+      .flatMap(exportProjectTriples(project, _))
+
+  private def trigExportFile(project: KnoraProject, tempDir: Path) = tempDir / s"${project.shortname}.trig"
```

Comment:
I don't know the requirements but I usually find it useful to have a timestamp appended to the filename. Like that, existing files (dumps in that case) won't be overridden.